New Single Cell Tutorial in R (#4141)
galaxyproject/training-material: topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio-WIP/preamble.md

# Introduction

You’ve previously done all the work to make a single cell matrix. Now it’s time to fully process our data using Seurat: remove low quality cells, reduce the many dimensions of data that make it difficult to work with, and ultimately try to define clusters and find some biological meaning and insights! There are many packages for analysing single cell data - Seurat (Satija et al. 2015), Scanpy (Wolf et al. 2018), Monocle (Trapnell et al. 2014), Scater (McCarthy et al. 2017), and many more. We’re working with Seurat in RStudio because it is well updated, broadly used and highly trusted within the field of bioinformatics.

> <comment-title></comment-title>
> This tutorial is significantly based on the [Seurat documentation](https://satijalab.com/seurat) as well as [Seurat's Guided Clustering Tutorial](https://satijalab.org/seurat/articles/pbmc3k_tutorial.html).
{: .comment}

## Get Data onto Galaxy 

> > Let's get our dataset loaded into Galaxy, first. 

Reviewer: ```suggestion
Let's get our dataset loaded into Galaxy, first. 
```

> <hands-on-title>GetData</hands-on-title>

> {% tool [EBI SCXA Data Retrieval](toolshed.g2.bx.psu.edu/repos/ebi-gxa/retrieve_scxa/retrieve_scxa/v0.0.2+galaxy2) %} with the following parameters: 

Reviewer: ```suggestion
> Run {% tool [EBI SCXA Data Retrieval](toolshed.g2.bx.psu.edu/repos/ebi-gxa/retrieve_scxa/retrieve_scxa/v0.0.2+galaxy2) %} with the following parameters: 
```

> - *"SC-Atlas experiment accession"*: 'E-MTAB-6945'
> - *"Choose the type of matrix to download"*: 'Raw filtered counts'
{: .hands_on}

> ## Open RStudio in Galaxy 
> You should now see a matrix.mtx, genes.tsv, barcodes.tsv, and exp_design.tsv files in your Galaxy history. For the rest of the workflow, let's move onto RStudio and get coding!
> > <hands-on-title>Open RStudio in Galaxy</hands-on-title>
> {% tool [RStudio](interactive_tool_rstudio)}
> {: .hands_on}

Reviewer: ```suggestion
## Open RStudio in Galaxy 

You should now see a matrix.mtx, genes.tsv, barcodes.tsv, and exp_design.tsv files in your Galaxy history. For the rest of the workflow, let's move onto RStudio and get coding!

> <hands-on-title>Open RStudio in Galaxy</hands-on-title>
> Run {% tool [RStudio](interactive_tool_rstudio)}
{: .hands_on}
```

Reviewer: adds more spacing for better legibility, but the important change is only the `> Run` before the tool link.

# Introduction

You’ve previously done all the work to make a single cell matrix. Now it’s time to fully process our data using Seurat: remove low quality cells, reduce the many dimensions of data that make it difficult to work with, and ultimately try to define clusters and find some biological meaning and insights! There are many packages for analysing single cell data - Seurat ({% cite Satija2015 %}), Scanpy ({% cite Wolf2018 %}), Monocle ({% cite Trapnell2014 %}), Scater ({% cite McCarthy2017 %}), and many more. We’re working with Seurat in RStudio because it is well updated, broadly used, and highly trusted within the field of bioinformatics.

> <comment-title></comment-title>

> > This tutorial is significantly based on the Seurat documentation({% cite Satija2015 %}) as well as [Seurat's Guided Clustering Tutorial](https://satijalab.org/seurat/articles/pbmc3k_tutorial.md).

Reviewer 2: this is unrelated to the linting error, but I think you want `.html` here
```suggestion
> This tutorial is significantly based on the Seurat documentation({% cite Satija2015 %}) as well as [Seurat's Guided Clustering Tutorial](https://satijalab.org/seurat/articles/pbmc3k_tutorial.html).
```

{: .comment}

> <agenda-title></agenda-title>
>
> In this tutorial, we will cover:
>
> 1. TOC
> {:toc}
>
{: .agenda}

> We’ll provided you with experimental data to analyse from a mouse dataset of fetal growth restriction [Bacon et al. 2018](https://usegalaxy.eu/training-material/topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.html#Bacon2018). This is the full dataset generated from [this tutorial](https://usegalaxy.eu/training-material/topics/single-cell/tutorials/scrna-case_alevin/tutorial.html). 

Reviewer 2: ```suggestion
We’ll provided you with experimental data to analyse from a mouse dataset of fetal growth restriction [Bacon et al. 2018]({% cite Bacon2018 %}). This is the full dataset generated from [this tutorial]({% link topics/single-cell/tutorials/scrna-case_alevin/tutorial.html %}). 
```

please use the `{% link topics/../../ %}` syntax for links to training materials.

And if you want to link to a reference/citation, please include it in this tutorial (so add a `tutorial.bib` file and put the citation in there (in this case you can copy it from the other tutorial's bib file)
